webpack/components/ActivationKeysSearch/index.js

```diff
@@ -23,24 +23,26 @@ import { selectAPIStatus } from 'foremanReact/redux/API/APISelectors';
 const ActivationKeysSearch = () => {
   const ACTIVATION_KEYS = 'ACTIVATION_KEYS';
   const KT_AK_LABEL = 'kt_activation_keys';
-  const selectedEnvId = useMemo(() => {
+  const getSelectedEnvId = () => {
```

Comment:
This seems to be vanilla JS, can we move it outside the component so the function isn't recreated on each render? That may provide the same benefit that the useMemo was supposed to.

Reply:
yeah, thats a better place for it